Mlem/Views/Tabs/Feeds/Components/CommentSortMenu.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CommentSortMenu.swift
+//  Mlem
+//
+//  Created by Sjmarf on 09/09/2023
+//
+
+import SwiftUI
+import Dependencies
+
+struct CommentSortMenu: View {
+    @Dependency(\.siteInformation) var siteInformation
+    
+    @Binding var isPresented: Bool
+    @Binding var selected: CommentSortType
+
+    var body: some View {
+        VStack {
+            VStack(spacing: 15) {
+                sortOption(.hot)
+                sortOption(.new)
+                sortOption(.old)
+                sortOption(.top)
+            }
+            Spacer()
+        }
+        .padding(20)
+        .presentationDetents([.height(300)])
+        .background(Color(UIColor.systemGroupedBackground))
+    }
+    
+    @ViewBuilder
+    private func sortOption(_ sortType: CommentSortType) -> some View {
+        Button {
+            isPresented = false
+            selected = sortType
+        } label: {
+            Label(sortType.label, systemImage: sortType.iconName)
+        }
+        .buttonStyle(SortMenuButtonStyle(isSelected: sortType == selected))
+        .accessibilityLabel(sortType.description)
+        .accessibilityHint("Double-tap to select")
```

Comment:
I don't know a ton about accessibility but do we normally bake these actions in like this? I imagine those gestures could be different per platform or something.

Reply:
Hmm, not sure. It's double-tap using Voice Over, but it might be different with other accessibility features. I'll remove these for now - the function of the buttons should be clear enough anyway, since the user probably knows they're in the sort mode picker anyway